modules/Robot.h

```diff
@@ -79,13 +82,14 @@ class Robot {
   bool getIsLeftEdge() const;
 
  private:
-  MotorController motorController;    // MotorControllerインスタンス
-  CameraCapture cameraCapture;        // CameraCaptureインスタンス
-  spikeapi::ColorSensor colorSensor;  // ColorSensorインスタンス
-  spikeapi::Clock clock;              // Clockインスタンス
-  spikeapi::Button button;            // Buttonインスタンス
-  spikeapi::ForceSensor forceSensor;  // ForceSensorインスタンス
-  spikeapi::Display display;          // Displayインスタンス
+  MotorController motorController;     // MotorControllerインスタンス
+  ICameraCapture* cameraCapturePtr;    // 実際に使うカメラ（抽象ポインタ）
+  CameraCapture defaultCameraCapture;  // 実機用のCameraCaptureインスタンス
```

Comment:
個人的には、null管理を一応考慮しないといけないポインタよりも参照の方が好みなのだけれど、ポインタじゃないとダメそうなヤツ？

```suggestion
  CameraCapture defaultCameraCapture;  // 実機用のCameraCaptureインスタンス (実体)
  ICameraCapture& cameraCapture;    // 実際に使うカメラ (参照)
```